<?xml version="1.0"?>
<launch>
	<arg name="ds4drv_cfg" default="$(find ds4ros)/cfg/ds4drv_example.conf" />
    <node pkg="ds4ros" type="ds4drv_wrapper.py" name="ds4_ros" output="screen">
		<param name="ds4drv_cfg" value="$(arg ds4drv_cfg)"/>
		<remap from="joy_node/dev" to="joy_node/dev" /> <!-- Remap this to the joy_node/dev parameter of your joystick node-->
    </node>
</launch>